definitions/npm/redux-rsaa-middleware_v1.x.x/flow_v0.66.x-/redux-rsaa-middleware_v1.x.x.js

```diff
@@ -0,0 +1,29 @@
+declare module 'redux-rsaa-middleware' {
+  import type { Middleware } from 'redux';
```

Comment:
This won't work, as imports aren't currently functional. You can create a non-exported type here which mocks `Middleware`. Something like:

```
type Middleware = (req: http$IncomingMessage, res: http$ServerResponse, next: (err?: ?Error) => mixed) => mixed;
```